pkg/cmd/cli/cli.go

```diff
@@ -34,9 +34,9 @@ To create a new application, you can use the example app source. Login to your s
 run new-app:
 
   $ %[1]s login
-  $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
+  $ %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-hello-world.git
```

Comment:
Just out of curiosity, are we planning to update those regularly? I understand that for tests the answer is - yes,  unless we introduce global constants to simplify that process? Or this is just one time issue, again does not apply to tests.